ank/src/cli_commands/get_state.rs

```diff
@@ -440,29 +115,32 @@ mod tests {
     // [utest -> swdd~cli-shall-support-desired-state-json~1]
     #[tokio::test]
     async fn utest_get_state_complete_desired_state_json() {
-        let test_data = test_utils::generate_test_complete_state(vec![
-            generate_test_workload_spec_with_param(
-                "agent_A".to_string(),
-                "name1".to_string(),
-                "runtime".to_string(),
-            ),
-            generate_test_workload_spec_with_param(
-                "agent_B".to_string(),
-                "name2".to_string(),
-                "runtime".to_string(),
-            ),
-            generate_test_workload_spec_with_param(
-                "agent_B".to_string(),
-                "name3".to_string(),
-                "runtime".to_string(),
-            ),
-        ]);
+        // TODO: replace this and all the other calls like this one
```

Comment:
There is a todo here.

Reply:
This should be treated with a specific conversion from one of the CompleteStates into a FilteredCompleteState.

Reply:
Alright, I'll handle this in my PR after this is merged in the ankaios branch